test/gui/tst_checkAlltabs/verificationPoints/Activity

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<VerificationPoint type="ObjectProperties" version="4">
+<Description/>
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="pos.x" type="int" valueType="Value"><![CDATA[195]]>
+<base64EncodedValue>MTk1</base64EncodedValue>
+</Verification>
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="pos.y" type="int" valueType="Value"><![CDATA[0]]>
+<base64EncodedValue>MA==</base64EncodedValue>
+</Verification>
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="text" type="QString" valueType="Value"><![CDATA[Activity]]>
+<base64EncodedValue>QWN0aXZpdHk=</base64EncodedValue>
+</Verification>
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="visible" type="bool" valueType="Value"><![CDATA[true]]>
```

Comment:
is there any way to get these objects from page objects instead of `names.py`?  🤔

Reply:
I don't think we can do that. They are auto generated when recording steps using verification point just like it is done here https://github.com/owncloud/client/blob/5211ff80696190e635fd9f7d77ddaf0074aaa569/test/gui/shared/verificationPoints/publicLinkExpirationProgressIndicatorInvisible#L4